tqdm/autonotebook.py

```diff
@@ -15,6 +15,7 @@
         raise ImportError("console")
     if 'VSCODE_PID' in os.environ:  # pragma: no cover
         raise ImportError("vscode")
+    import ipywidgets
```

Comment:
This PR seems like a good fix overall, but it needs this to pass CI checks:
```suggestion
    import ipywidgets  # noqa: F401
```